client/my-sites/upgrades/domain-management/name-servers/dns-template-button-group.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+
+class DnsTemplateButtonGroup extends Component {
```

Comment:
You're gonna hate me for suggesting this, but I've noticed there's a component that fits this use case even better: [`SegmentedControl`](http://calypso.localhost:3000/devdocs/design/segmented-control). I especially like its feature of showing which part is "active"/selected, something which is IMHO missing from the ButtonGroup.
I think spreading the component to full width looks better, too:
<img width="749" alt="screen shot 2017-05-15 at 15 14 25" src="https://cloud.githubusercontent.com/assets/3392497/26059369/d935d432-3981-11e7-9f02-d88c19e2b426.png">

But IANAD (I am not a designer ;)), so treat these with a grain of salt :D